Symfony/CS/Tests/DocBlock/TagTest.php

```diff
@@ -37,6 +37,7 @@ public function provideNameCases()
             array('THROWSSS', "\t@THROWSSS\t  \t RUNTIMEEEEeXCEPTION\t\t\t\t\t\t\t\n\n\n"),
             array('other', ' *   @\Foo\Bar(baz = 123)'),
             array('expectedException', '     * @expectedException Exception'),
+            array('property-read', ' * @property-read integer $daysInMonth number of days in the given month'),
```

Comment:
1.10 doesn't support any tags with `-`, so this PR won't fix behaviour of any fixer